### DISCUSSION TITLE
setup.py cannot be run help me.

### DISCUSSION THREAD

**Discussion by Yangjaehong** [inducer/pycuda#425] [Q&A]
In setup.py, run setup.py to change Switch("CUDA_ENABLE_GL", True, "Enable CUDA GL interoperability") to True.
But I don't know why it doesn't work because of the environment variables.
I used conda.
Please help me.

`C:\Users\dfae\Downloads\pycuda-2022.2.2\src\cpp\cuda.hpp(23): fatal error C1083: 포함 파일을 열 수 없습니다. 'cudaProfiler.h': No such file or directory
error: command 'C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.36.32532\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
`
window 11
python : 3.8.18
cuda : 11.6
torch : 1.13.1

**Comment by inducer** [comment]
- Is `cudaProfiler.h` present anywhere on your system? (It should be if CUDA is installed properly.)
- If yes, did you configure the CUDA include path (using `./configure.py`) to let PyCUDA know where the headers live?

**Reply by Yangjaehong** [reply]
1. cuda is installed. I've used it well before.
2. i have not set up configure.py. Can you tell me how to configure it in ./configure.py?

**Reply by Yangjaehong** [reply]
Solved. See configure --help. thank you.
